src/callbacks_step/save_solution.jl

```diff
@@ -211,11 +211,11 @@ end
         get_element_variables!(element_variables, u_ode, semi)
         callbacks = integrator.opts.callback
         if callbacks isa CallbackSet
-            for cb in callbacks.continuous_callbacks
+            foreach(callbacks.continuous_callbacks) do cb
```

Comment:
I always wondered why the save solution callback is so expensive (in the sense of allocations)
I guess this explains it!